sessions/main.py

```diff
@@ -0,0 +1,68 @@
+# Copyright 2019 Google LLC All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import random
+from uuid import uuid4
+
+from google.cloud import firestore
+from flask import Flask, make_response, request
+
+
+app = Flask(__name__)
+db = firestore.Client()
+sessions = db.collection('sessions')
+colors = ['red', 'blue', 'green', 'yellow', 'pink']
```

Comment:
We decided to use greetings in different languages instead of background colors for accessiblity reasons. See e.g., https://cloud.google.com/nodejs/getting-started/session-handling-with-firestore